frontend/src/app/api/locationEntity.service.ts

```diff
@@ -819,4 +819,4 @@ export class LocationEntityService {
         );
     }
 
-}
+}
```

Comment:
Had trouble getting location entity service to not have any changes. Sigh. Ignore this file, this is legit copy and pasted from what's in master.